pact/dao/dao.pact

```diff
@@ -0,0 +1,279 @@
+(enforce-pact-version "3.7")
+
+(namespace (read-msg 'dao-ns))
+
+(module utils-v1 GOVERNANCE
+  (defcap GOVERNANCE () false)
```

Comment:
Just a note that you have to `(enforce false "...")` to fail governance. `false` is insufficient.